src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -8,6 +8,7 @@ Release History
 * vm create: Added --computer-name for setting a vm's computer name.
 * vm/vmss create: `--ssh-key-value` renamed to `--ssh-key-values` and can now accept multiple ssh public key values or paths.
   Note: this is **not** a breaking change. `--ssh-key-value` will be parsed correctly as it matches only `--ssh-key-values`
+* ppg create: `--type`, the proximity placement group type, is now optional.
```

Comment:
this change should go to top

Reply:
Isn’t this change with these other changes that were merged recently? 

Are you saying I should move it above 2.2.21 or to be the first point within 2.2.21

Reply:
@adewaleo, sorry I was wrong, because I didn't realize you have already made a few changes since //build.

Reply:
No worries @yugangw-msft! Yes these changes are post-build!